.travis.yml

```diff
@@ -7,12 +7,10 @@ matrix:
   fast_finish: true
   include:
     - env: DM=~6.0
-      php: 5.3
-    - env: DM=~6.0
-      php: 5.4
-    - env: DM=@dev
-      php: 5.5
+      php: 7.1
     - env: DM=~5.0
+      php: 5.5
+    - env: DM=@dev
       php: 5.6
```

Comment:
You switched the two DataModel versions PHP 5.5 and 5.6 use. Not that it matters, just wondering. A special reason to do this?